packages/custom-functions-metadata/src/parseTree.ts

```diff
@@ -97,6 +117,7 @@ interface IJsDocParamType {
 }
 
 const CUSTOM_FUNCTION = "customfunction"; // case insensitive @CustomFunction tag to identify custom functions in JSDoc
+const CUSTOM_ENUM = "customenum"; // case insensitive @CustomEnum tag to identify custom enums in JSDoc
```

Comment:
nit: put these constants in alphabetical order.

Reply:
Updated the tag strings alphabetically